core/client/client.go

```diff
@@ -77,8 +76,6 @@ type VerifierInterface interface {
 	// VerifyRevision verifies that revision is correctly signed and included in the append only log.
 	// VerifyRevision also verifies that revision.LogRoot is consistent with the last trusted SignedLogRoot.
 	VerifyRevision(revision *pb.Revision, trusted types.LogRootV1) (*types.LogRootV1, *types.MapRootV1, error)
-	// VerifySignedMapRoot verifies the signature on the SignedMapRoot.
-	VerifySignedMapRoot(smr *trillian.SignedMapRoot) (*types.MapRootV1, error)
```

Comment:
Do you still need the implementations of this method?

Reply:
We don't, and it sort of doesn't exit in the sense that it's a property of the Trillian map verifier which the verifier inherits.  I can pull it out to make it more explicit if you'd like?